vue.config.js

```diff
@@ -15,7 +15,8 @@ module.exports = {
 								'x64'
 							]
 						}
-					]
+					],
+					icon: 'build/icon.ico'
```

Comment:
Isn't specifying this redundant? Also the subsequent calls?

Reply:
It's because of the extension. Because Linux wants a png I moved the ico to Windows only.